lib/core/notification/notification_service.dart

```diff
@@ -4,12 +4,34 @@ import 'package:firebase_messaging/firebase_messaging.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter_local_notifications/flutter_local_notifications.dart';
 import 'package:givt_app/app/injection/injection.dart';
+import 'package:givt_app/app/routes/app_router.dart';
+import 'package:givt_app/app/routes/routes.dart';
 import 'package:givt_app/core/logging/logging_service.dart';
 import 'package:givt_app/shared/repositories/givt_repository.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 
 import 'package:timezone/timezone.dart' as tz;
 
+/// Navigate to the screen based on the notification payload
+/// This is called when the user taps on the notification
+@pragma('vm:entry-point')
+Future<void> _navigateToScreen(NotificationResponse details) async {
+  final payload = details.payload;
+  switch (payload) {
+    case 'RecurringDonation':
+      await LoggingInfo.instance
```

Comment:
Maybe its good to implement them or leave the logging out. 
Right now we would get logging in Logit, but the action will never happen